cmake/CMakeLists.txt

```diff
@@ -18,5 +18,4 @@ set(
     )
 mark_as_advanced(GLAD_CMAKE_DIR)
 
-list(APPEND CMAKE_PREFIX_PATH ${GLAD_SOURCES_DIR})
-find_package(Glad REQUIRED)
+include(GladConfig.cmake)
```

Comment:
I don't think this is necessary, as as far as I know all paths such as these are relative, but I've replaced it with `${CMAKE_CURRENT_SOURCE_DIR}/GladConfig.cmake` for good measure.

Reply:
https://cmake.org/cmake/help/latest/command/include.html

`include` can work with `module` if the filename is like `<modulename>.cmake`, and will use `CMAKE_MODULE_PATH`.

Also https://cmake.org/cmake/help/latest/policy/CMP0017.html

I am not sure how it will be applied.

But relative path is more consistent with previous behavior.